src/osdc/Objecter.h

```diff
@@ -1154,7 +1154,7 @@ struct ObjectOperation {
     for (uint32_t i = 0; i < sops.size(); i++) {
       out_bl[i] = &sops[i].outdata;
       out_handler[i] = NULL;
-      out_rval[i] = &sops[i].rval;
+      out_rval[i] = (int*)(&sops[i].rval);
```

Comment:
@liewegas 
Right, that si more to the point.
The debugging code cruft is removed.